nexus/src/external_api/http_entrypoints.rs

```diff
@@ -4401,6 +4402,30 @@ async fn uninitialized_sled_list(
     };
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
+/// Add a sled to an initialized rack
```

Comment:
```suggestion

/// Add a sled to an initialized rack
```

Reply:
Fixed in [61db1c9](https://github.com/oxidecomputer/omicron/pull/4545/commits/61db1c97e856104c65291dee8651069a677b1a7d)